src/test/suite/fixtures/recorded/actions/copyVest.yml

```diff
@@ -32,5 +32,5 @@ finalState:
         start: {line: 1, character: 6}
         end: {line: 1, character: 11}
       isReversed: false
-      hasExplicitRange: true
```

Comment:
Why was this true before?

Reply:
https://github.com/cursorless-dev/cursorless/pull/965#discussion_r973741366